### DISCUSSION TITLE
Is there any way to get the list of options for a select field?

### DISCUSSION THREAD

**Discussion by sergio9929** [pocketbase/pocketbase#2912] [Q&A]
I was wondering if there is any way to get the list of options for a select field with the js-sdk. It would be very helpful when creating forms.

Thank you in advance.

**Comment by ganigeorgiev** [comment]
No, at least not directly. You can fetch the entire collection configuration model by using the [Collections web APIs](https://pocketbase.io/docs/api-collections/) but they require to be authenticated as admin.

If you have a lot of select options, then it may be better to move the options to their own collection and change the `select` field to `relation`.

**Comment by sergio9929** [comment]
Thank you for the quick response.
